apple/Sources/FerrostarMapLibreUI/Views/NavigationMapView.swift

```diff
@@ -24,13 +27,13 @@ public struct NavigationMapView: View {
         darkStyleURL: URL,
         navigationState: NavigationState?,
         camera: Binding<MapViewCamera>,
-        distanceFormatter: Formatter = MKDistanceFormatter()
+        @MapViewContentBuilder _ makeMapContent: () -> [StyleLayerDefinition] = { [] }
```

Comment:
What happened to the distance formatter? We definitely want to keep that user- configurable.

Reply:
This is only a "MapView" now, so there's nothing to format here. It's all pushed up one later to the "Navigation Views" where the instruction view is included.